Makefile

```diff
@@ -0,0 +1,3 @@
+copy:
+	cp ./src/spritesh ./bin/npm/spritesh
+	cp ./src/spritesh ./bin/gem/sprite.sh
```

Comment:
Yep, came across this. Could you have a look at https://github.com/edenspiekermann/sprite.sh/pull/8#issuecomment-186163044 and tell me your opinion?